src/webgpu/util/f32_interval.ts

```diff
@@ -47,11 +41,10 @@ export class F32Interval {
 
   /** @returns if a point or interval is completely contained by this interval
    *
-   * Due to the fact backends may clamp out of range values to the min/max f32
-   * values, which is represented in the implementation by extending the
-   * begin/end values as appropriate, there some unintuitive behaviours here.
-   * For example:
-   *   [0, max f32].contains(+∞) will return true.
+   * Due to values that are above/below the f32 range being indistinguishable
+   * from other values out of range in the same way, there some unintuitive
+   * behaviours here, for example:
+   *   [0, max f32 +1].contains(+∞) will return true.
```

Comment:
This should be something like `[0, greater than max f32]` or `[0, 2 * max f32]`, because technically `max f32 + 1` rounds to `max f32` due to FP precision being low on the edges